packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -331,6 +338,13 @@ export interface ICompressionRuntimeOptions {
 
 /**
  * Options for container runtime.
+ *
+ * @privateRemarks If any new properties are added to this interface (or IContainerRuntimeOptionsInternal), then we will also need
+ * to make changes in compatUtils.ts.
+ * If the new property is not considered version-dependent, then it must be explicity omitted from IContainerRuntimeOptionsVersionDependent.
+ * If it is considered version-dependent, then a corresponding entry must be added to `versionDependentOptionConfigMap` with the appropriate compat
```

Comment:
If this is not followed, then something will break right? Maybe add that info here so devs would know exactly what to do.